llvmlite/tests/test_ir.py

```diff
@@ -2064,6 +2064,10 @@ def test_vector(self):
             ', '.join(map('i32 {}'.format, vals)))
         self.assertEqual(str(vec), vec_repr)
 
+    def test_non_nullable_int(self):
+        constant = ir.Constant(ir.IntType(32), None).constant
+        assert constant == 0
```

Comment:
```suggestion
        self.assertEqual(constant, 0)
```
`llvmlite` testing uses unittest assertions